docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configure Teleport to create host users
+description: How to configure Teleport to automatically create transient host users.
+---
+
+Teleport's SSH service can be configured to automatically create local Unix users
+upon login.
+
+Host users created by Teleport are transient and will be deleted at the end of an SSH
+session.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+The following utilities should be available in the PATH:
+- useradd
+- userdel
+- usermod
+- groupadd
+- getent
+- visudo
+
+## Role configuration
+
+First, a role must be created with `create_host_user: true` in order to allow users
+in this role to have host users created for them.
+
+Example:
+```yaml
+kind: role
+version: v5
+metadata:
+  name: auto-user-groups
+spec:
+  options:
+    # allow automatic creation of users.
+    create_host_user: true
+  allow:
+    # username from external attribute.
+    logins: [ "{{external.unixLogin}}" ]
+    # list of groups the created user should be added to.
+    groups: [ubuntu, nginx, other]
+    # list of entries to include in a temporary sudoers file created in /etc/sudoers.d
+    host_sudoers: [
+       "{{internal.logins}} ALL = (root) NOPASSWD: /usr/bin/systemctl restart nginx.service"
+    ]
+    node_labels:
+      - 'env': 'devel'
+```
+
+NOTE: each of the user `logins` entries, should conform to the username requirements
+of the Linux distribution being used. Distribution username requirements: https://systemd.io/USER_NAMES/
+
+NOTE: All Teleport user roles matching a node **must** include `create_host_user: true` in
```

Comment:
If I have a user called `myuser` with roles `allow-dev` and `allow-all`, `allow-dev` has an `allow` rule with `node_labels: {'env': 'dev'}`, and `allow-all` has an `allow` rule with `node_labels:{'*':'*'}`, then for a Node with label `env:dev`, does this mean that both `allow-all` and `allow-dev` need `create_host_user: true` to be true for automatic user creation to be enabled? 

What if the user also has a role with `create_host_user: false` that does not match the Node?